waku/v2/protocol/waku_store/waku_store.nim

```diff
@@ -110,16 +110,29 @@ proc init*(T: type PagingInfo, buffer: seq[byte]): ProtoResult[T] =
   pagingInfo.direction = PagingDirection(direction)
 
   ok(pagingInfo) 
-  
+
+proc init*(T: type HistoryContentFilter, buffer: seq[byte]): ProtoResult[T] =
+  let pb = initProtoBuffer(buffer)
+
+  var topic: ContentTopic
```

Comment:
Ditto here, I'd be more explicit with contentTopic or even "ct" would be better than "topic"

Reply:
Sure, addressed.